nilearn/datasets/atlas.py

```diff
@@ -351,26 +440,31 @@ def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
     opts = {'uncompress': True}
     root = os.path.join('data', 'atlases')
 
-    if atlas_name[0] == 'c':
-        if 'cort-maxprob' in atlas_name and symmetric_split:
-            split_name = atlas_name.split('cort')
-            atlas_name = 'cortl' + split_name[1]
-            label_file = 'HarvardOxford-Cortical-Lateralized.xml'
-            lateralized = True
-        else:
-            label_file = 'HarvardOxford-Cortical.xml'
+    if source_idx == 0:
+        if atlas_name[0] == 'c':
             lateralized = False
+            if 'cort-maxprob' in atlas_name or 'cortl-maxprob' in atlas_name and symmetric_split:
+                split_name = atlas_name.split('cort')
+                atlas_name = 'cortl{}'.format(split_name[1])
+                label_file = 'HarvardOxford-Cortical-Lateralized.xml'
+                lateralized = True
```

Comment:
Yes. This was an old part of the code. But because HarvardOxford maps now supports the lateralized atlases, so I joined cortl with (cort + symm) in the condtition